rules/react.js

```diff
@@ -29,9 +29,6 @@ module.exports = {
         // to make use of them, and when not; see https://github.com/Expensify/Style-Guide/pull/60 for more details
         'react/destructuring-assignment': 'off',
 
-        // This rule is broken for React components created using `window.CreateClass`, so we disable it temporarily
-        'react/no-this-in-sfc': 'off',
```

Comment:
Do we do this often?

Reply:
`window.createClass` is used quite a bit in our older code, yeah. It's from earlier versions of React before ES6 classes were supported.